src/pages/patientView/clinicalInformation/CopyNumberCountCache.ts

```diff
@@ -21,7 +21,7 @@ function fetch(queries:CopyNumberCountIdentifier[], geneticProfileIdDiscrete:str
     }
 }
 
-export default class CopyNumberCountCache extends LazyMobXCache<CopyNumberCount, CopyNumberCountIdentifier> {
+export default class CopyNumberCountCache extends LazyMobXCache<CopyNumberCount, CopyNumberCountIdentifier, any> {
```

Comment:
it seems like in all the uses of the cache, the type of Meta is any, so why does it need to be specified? (apologies if i'm missing the exception!)